metrics/web/docs/features/remote_config/diagrams/remote_config_domain_layer_class_diagram.puml

```diff
@@ -0,0 +1,39 @@
+@startuml remote_config_domain_layer_class_diagram
+
+package base.domain.usecases {
+    interface UseCase<Type, Params> {
+        call: Type
+    }
+}
+
+package common.domain {
+    package repository {
+        interface RemoteConfigurationRepository {
+            + void applyDefaults();
```

Comment:
```suggestion
            + void setDefaults(Map<String, dynamic>);
```
But it seems like we are exposing fields of entities - we should recheck that in other documents. In that case: 
```suggestion
            + void setDefaults(isLoginFormEnabled, isFpsMonitorEnabled, isRendererDisplayEnabled);
```